### DISCUSSION TITLE
Specify github subfolder to run copy

### DISCUSSION THREAD

**Discussion by AndreSepulveda** [copier-org/copier#1861] [Q&A]
Basically I have multiple templates within the same github repo and Im trying to specify from which subfolder copier should run the copy command. Not sure if I missed something or if it's really not a default option. 

If it's not possible, I believe it would be a nice feature to add since more people might want to use the same repo to hold multiple templates.

My structure is like this and I'd want to run `copier copy git@my-repo ./my-target  -folder templates/template_1`

```
root/
  |---templates/
              |----- template_1/
              |               |-----  copier.yml
              |               |-----  main.py
              |
              |----- template_2/
                              |-----  copier.yml
                              |-----  main.py
```
  
 Appreciate any help/clarification with this

**Comment by sisp** [comment]
This is answered in the docs: https://copier.readthedocs.io/en/latest/configuring/#subdirectory

> **Can I have multiple templates in a single repo using this option?**
>
> The Copier recommendation is: **1 template = 1 Git repository.**
>
> [...]

Looks like a duplicate of #315 #1078 #1279 #1571. See especially https://github.com/copier-org/copier/issues/315#issuecomment-746007580 and https://github.com/copier-org/copier/issues/1571#issuecomment-2423035861.

I'm closing this discussion, but feel free to continue commenting.

**Reply by AndreSepulveda** [reply]
So it's not supported. Thanks

**Reply by sisp** [reply]
No, it isn't.
